server/drivers/clickhouse/_clickhouse.js

```diff
@@ -37,7 +37,7 @@ function send(config, query) {
     }
   )
     .then((response) =>
-      response.ok ? response.json() : { error: response.text() }
+      response.ok ? response.text() : { error: response.text() }
```

Comment:
I notice this is getting JSON.parsed later on - is there a reason for using `.text()` instead of `.json()`?

Reply:
Yeah, when executing insert/create etal. queries, clickhouse will return empty response, that result in the JSON parse error. This is the main reason to use `.text()` function.

Reply:
Ah ok. yeah that always gets tricky.